web_widget_tolocalestring/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Web Widget toLocaleString
+==============
+
+This module add 'tolocalestring' widget in tree view for formatting numbers or not to display the values 0 : 
+
+
+.. image:: /web_widget_tolocalestring/static/description/screenshot1.png
+    :alt: Screenshot
+
+
+
+Installation
+============
+
+It was tested on Odoo trunk, 8.0 branch.
+
+
+
+
+Usage
+=====
+
+To use this module, you need in the view declaration, put widget='tolocalestring' and tolocalestring='...' attribute in the field tag::
```

Comment:
To use this module, you need to add inside the view declaration widget='tolocalestring' and tolocalestring='...' attribute such as: